package/libyui-qt-graph.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Thu Nov 28 12:51:31 CET 2019 - aschnell@suse.com
+
+- Fixed unescaping of graphviz escString (bsc#1157916)
```

Comment:
That is correct but requires the reader to perform a web lookup of "graphviz escString".
Let's spill the details right away:

```suggestion
- Backslash escapes in labels are respected now
  (graphviz escString, bsc#1157916)
```